logbook-api/src/main/java/org/zalando/logbook/StructuredHttpLogFormatter.java

```diff
@@ -91,14 +94,17 @@ default Map<String, Object> prepare(final Correlation correlation, final HttpRes
         content.put("protocol", response.getProtocolVersion());
         content.put("status", response.getStatus());
 
+        if (!response.getAttributes().isEmpty())
+            content.put("attributes", response.getAttributes());
```

Comment:
This actually introduces a behavior that differs from the README and isn't specified in any test.
The JSONHttpLogFormatter will try to serialize the attributes as their original types (because we pass them as Object.
So the `toString` wouldn't be used, as the README claims it.

This is an interesting aspect, because it might be nice to get fully typed JSON values for non-primitive attributes, but it's very subtle right now, because there is no documentation nor test as far as I can tell, that verifies this.

Reply:
Fixed in 0725b94: It's now bothe documented and tested.